doc/stdlib/parallel.rst

```diff
@@ -374,6 +377,11 @@ General Parallel Computing Support
             body
         end
 
+.. function:: @everywhere
+
+    Executes an expression as is on all processors. In the event of exceptions on
```

Comment:
"as is" -> I was being lazy, It means the expression is not wrapped in a closure, i.e., `localize_vars` is not called on it. 

I can only explain this in opposition to `@async` and `@spawn` - "Unlike `@async` and `@spawn`, which create closures around the expression and execute them, `@everywhere` executes the expression as is on all processes"

Any better suggestions?